acquia/acquia_cms
# ACMS-1711: import logo after profile installation completed.

**Motivation**
Fixes #ACMS-1711

**Proposed changes**
import logo after profile installation completed.
Update modules to latest version, add conflict with Drupal 10, and remove install_acms_import_default_content task as default content removed from ACMS Image.

**Alternatives considered**
NA

**Testing steps**
- Checkout ACMS-1711-1.5.x branch using `git clone --branch ACMS-1711-1.5.x git@github.com:acquia/acquia_cms.git`
- Install all dependencies using `composer install && composer update --lock`
- Install site with this code and you should be able to install without any issue(after ACMS Image module release i.e. 1.5.11) using `./vendor/bin/drush si acquia_cms --account-pass=admin`.
- Meanwhile ACMS Image module gets released, copy paste code from [ACMS-1711](https://github.com/acquia/acquia_cms/tree/ACMS-1711) branch for ACMS image module.
- Install site using `./vendor/bin/drush si acquia_cms --account-pass=admin `
- Verify image media created at `/admin/content/media`. 
- Export config using `./vendor/bin/drush cex -y`
- Get site uuid using `./vendor/bin/drush cget system.site uuid`
- Copy the uuid
- **Site install with existing site:**
- Install site using `./vendor/bin/drush si acquia_cms --account-pass=admin --existing-config && drush cr`
- Verify image media created at `/admin/content/media`. 
- **Optional**
- Install site using `./vendor/bin/drush si acquia_cms --account-pass=admin && ./vendor/bin/drush cset system.site uuid <copied-uuid> -y && ./vendor/bin/drush cim -y && ./vendor/bin/drush cr`
- Verify image media created at `/admin/content/media`. 

**Merge requirements**
